UPGRADE-3.1.md

```diff
@@ -24,6 +24,22 @@ FrameworkBundle
 
  * As it was never an officially supported feature, the support for absolute
    template paths has been deprecated and will be removed in Symfony 4.0.
+ * Since the refactoring of 2.8, form types without dependencies are autoloaded
+   by the `DependencyInjectionExtension` in the form registry.
+   Consequently they are deprecated and will be removed in 4.0.
```

Comment:
OK I got your point, so it should be:
```
Since the refactoring of 2.8, form types without dependencies are autoloaded
by the `DependencyInjectionExtension` only, through the form registry. So they
do not require to be declared as services anymore.
Consequently the services corresponding to these form types are deprecated
and will be removed in 4.0.
```
?